test/fixtures/service_fixtures.js

```diff
@@ -287,6 +287,25 @@ module.exports = {
     }
   },
 
+  validUpdateRequestToGoLiveRequest: (opts) => {
+    opts = opts || {}
+
+    const data = {
+      op: opts.op || 'replace',
+      path: opts.path || 'current_go_live_stage',
+      value: opts.value || 'ENTERED_ORGANISATION_NAME'
+    }
```

Comment:
Could you make this fixture stricter by only passing in the `value` as a parameter?
We should then use this fixture directly in the cypress tests for stubbing the update go live stage requests.

This will give us confidence that making a patch request to the endpoint with the path `current_go_live_stage` works, and if this changes the Cypress tests won't get out of date.